python/ray/dataframe/utils.py

```diff
@@ -305,4 +310,17 @@ def _reindex_helper(df, old_index, new_index, axis):
         df.columns = old_index
         df = df.reindex(columns=new_index, copy=False)
         df.columns = pd.RangeIndex(len(df.columns))
-    return df
+    return create_blocks_helper(df, npartitions, axis)
+
+
+@ray.remote
+def co_op_helper(func, left_columns, right_columns, left_df_len, *zipped):
```

Comment:
Is `co` short for "column"? May be worth clarifying this. This function could probably use a docstring.

Also how do you determine which function names in this file should start with `_`?

Reply:
`co op` is short for `copartition operation`. I will clarify. The note about the leading underscore is something I have been wanting to clean up. This should lead with an underscore.